CHANGELOG.md

```diff
@@ -25,6 +25,7 @@
 
 ### Fixes
 
+- Fixed checkbox tick offset ([PR 2751](https://github.com/input-output-hk/daedalus/pull/2751))
```

Comment:
you need to merge develop into your branch and move this line on the vNext section